File: 240409/폭탄 해제/bomb-removal.py
code, color, second = input().split()
second = int(second)

class CODE:
    def __init__(self, code, color, second):
        self.code = code
        self.color = color
        self.second = second

    def print_code(self):
        print("code :", self.code)
        print("color :", self.color)
        print("second :", self.second)

code_inst = CODE(code, color, second)
code_inst.print_code()

File: 260101/폭탄 해제/bomb-removal.cpp
#include <iostream>
#include <string>

using namespace std;

class CODE 
{
    public:
        string code;
        string color;
        int second;

        CODE(string c, string col, int s)
            : code(c), color(col), second(s) {}

        void print_code() {
            cout << "code : " << code << endl;
            cout << "color : " << color << endl;
            cout << "second : " << second << endl;
        }
};

int main() 
{
    string code, color;
    int second;

    cin >> code >> color >> second;

    CODE code_inst(code, color, second);
    code_inst.print_code();

    return 0;
}
